
public class Exceptions {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		int b=7;
		int c=0;
		
//		try
		{
			//purchased faile
			int k=b/c;//
			
			System.out.println(k);
		}
  //   catch(Exception e)
		{
    	 //retry again
    //	 System.out.println("I catch error exception");
		}
	}

}
